### DISCUSSION TITLE
How to search every wiki/file content I have downloaded in kiwix-desktop?

### DISCUSSION THREAD

**Discussion by Cloufish** [kiwix/kiwix-desktop#952] [Q&A] [answered]
tldr: 
I just want to use search bar as "Google" (I know it's not appropriate term xd)

When I'm searching for Physics Topic I'd want to get result from both normal Wikipedia and "Physics by Wikipedia", Khan Academy and more.

If it's not possible, why do you include indices in .zim files (as stated here -> https://github.com/kiwix/kiwix-tools/issues/242)
Instead of _additionally_  providing a raw html files for people to index the contents themselves. (I'm not saying to completely abandon .zim)

**Comment by kelson42** [comment, marked as answer]
@Cloufish What you aim for is called by us "multizim search". It's not possible for the moment, you can only make one zim file search at a time. But this feature request is quite high on the feature list, look at https://github.com/openzim/libzim/issues/734 and https://github.com/kiwix/kiwix-desktop/issues/315.

**Reply by Cloufish** [reply]
Would be great to have this.

Because when it comes to self-hosted homelab,
it's very easy to index files on the drive (movies, books, etc.)

But very hard to do that on .zim files.
Having at least this feature would make experience so much better

**Reply by geheimniskonto** [reply]
You might find this helpful...
github.com/mozanunal/llm-tools-kiwix

I plan to test the approach across my universe of recoll and kiwix 'datasets'.
